src/Generator.php

```diff
@@ -1047,6 +1047,10 @@ private function isReturnTypeFinal(ReflectionMethod $method)
 
         $returnType = (string) $method->getReturnType();
 
+        if ($returnType === \Generator::class || $returnType === \Closure::class) {
```

Comment:
With this `if` this method won't actually do what its name says.

Reply:
Maybe `canReturnTypeBeStubbed()` would be a better name now.